pkg/cluster/storage.go

```diff
@@ -22,8 +23,21 @@ func (c *Cluster) save(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
+	storageClient, err := client.New(c.etcdConfig)
+	if err != nil {
+		return err
+	}
+	c.storageClient = storageClient
```

Comment:
We'd talked about removing storageClient from the Cluster struct, since save and storageBootstrap are the only two users of it, and both of them will create a new client when they are called.